api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -24,7 +24,7 @@ import (
 	routev1 "github.com/openshift/api/route/v1"
 	autoscalingv1 "k8s.io/api/autoscaling/v1"
 	"k8s.io/api/core/v1"
-	"k8s.io/api/extensions/v1beta1"
+	networkingv1 "k8s.io/api/networking/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	runtime "k8s.io/apimachinery/pkg/runtime"
```

Comment:
It looks like there is spacing difference between edited code and generated code.   Run `make` and  this file is re-generated with differences.  
![image](https://user-images.githubusercontent.com/44041377/132113191-886648db-f67b-44b8-9360-e5b16fbc36ed.png)